lib/kg/sales.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Kg.Sales do
+  alias Kg.Sales.Sale
+  alias Kg.Repo
+
+  import Ecto.Query, warn: false
+
+  def create_sale(attrs) do
+    %Sale{}
+    |> Sale.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def all_sales() do
+    query = from(c in Sale)
```

Comment:
typically we use the first character of the model name... so the standard would be `from(s in Sale)`